app.py

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env python3
 import os
 import boto3 
+import string
+import random
 import aws_cdk as cdk
 import aws_cdk.assertions as assertions
 from sds_in_a_box.sds_in_a_box_stack import SdsInABoxStack
 
-
+### Implement the code found in "sds_in_a_box_stack.py"
```

Comment:
```suggestion
```
I'm not sure what the comment means. Maybe as a module-level docstring instead?

Reply:
I'm not sure either tbh